static/app/utils/discover/styles.tsx

```diff
@@ -51,3 +52,11 @@ export const UserIcon = styled(IconUser)`
   margin-left: ${space(1)};
   color: ${p => p.theme.gray400};
 `;
+
+export const ActorContainer = styled('div')`
+  display: flex;
+  justify-content: center;
+  :hover {
+    cursor: pointer;
```

Comment:
Will there be functionality later to be able to click and change the assignee from the widget?
1, the pointer cursor makes me think I can do something but it doesn't do anything right now
2, changing the assignee from the widget would be pretty sweet

Reply:
👍  Confirmed we probably won't be adding functionality to the assignee badges for now. Update the `cursor` to `default` since the auto setting uses the text highlighter which just looks weird.